### PR TITLE
Move PDB controller and type ownership to SIG-Apps

### PR DESCRIPTION
**Pull request by erictune**
Implement https://github.com/kubernetes/features/issues/285#issuecomment-298703241

### CODE REVIEW

File: pkg/apis/policy/OWNERS [hunk 1]
```diff
@@ -1,14 +1,4 @@
...
-- caesarxuchao
-- sttts
-- ncdc
-- timothysc
-- dims
-- mml
-- mbohlool
-- david-mcmahon
-- jianhuiz
+- sig-pr-reviews
```
Comment (ncdc) [inline pkg/apis/policy/OWNERS]: missing -apps-?

File: pkg/controller/disruption/OWNERS [hunk 2]
```diff
@@ -1,5 +1,2 @@
 reviewers:
-- mml
-- davidopp
-- kargakis
-- mwielgus
+- sig-apps-pr-reviewers
```
Comment (ncdc) [inline pkg/controller/disruption/OWNERS]: should be sig-apps-pr-reviews (not reviewers)

Comment (erictune) [inline pkg/controller/disruption/OWNERS]: Thanks Andy, fixed.

**Comment by krmayankk** [comment]
@erictune asking here, since i dont know where else to ask. How can i add myself as a reviewer for certain controllers . Do i need permission from someone to be added ?

**Comment by erictune** [comment]
@krmayankk I'm looking into what the process is.  It isn't documented well.

**Comment by kargakis** [comment]
@erictune shouldn't the newly added groups in the ONWERS files be aliased [here](https://github.com/kubernetes/kubernetes/blob/master/OWNERS_ALIASES)?

**Comment by erictune** [comment]
PR Updated to use OWNER_ALIASES.
The use of that file is now documented in https://github.com/kubernetes/test-infra/pull/2562/files#diff-b59baac18a964b06d0f147ebc5f3c8cc

**Comment by erictune** [comment]
@krmayankk as best I can tell, the process is do 20 PRs and be invited by Approvers.
This is from: https://github.com/kubernetes/community/blob/master/governance.md#code-and-documentation-contributors

**Comment by erictune** [comment]
@k8s-bot verify test this

**Comment by erictune** [comment]
@krmayankk Also, you should feel free to do code reviews on any PR in an area that interests you, even if you are not an official reviewer.

**Comment by krmayankk** [comment]
@erictune thanks. I want to be added to important areas so that i get notifications on new PR. I think i have so far done more than 20 PRs and submitted a couple PR as well.

**Comment by erictune** [comment]
@k8s-bot tell me a joke

**Comment by erictune** [comment]
@k8s-bot verify test this

**Comment by davidopp** [comment]
@k8s-bot gce etcd3 e2e test this
@k8s-bot verify test this
/lgtm

File: OWNERS_ALIASES [hunk 3]
```diff
@@ -64,4 +64,19 @@ aliases:
...
+    - erictune
+    - foxish
+    - janetkuo
+    - kargakis
+    - kow3ns
+    - lukaszo
+    - mfojtik
+    - smarterclayton
+    - soltysh
+  sig-apps-api-reviewers:
```
Comment (kargakis) [inline OWNERS_ALIASES]: Does not match what you add in pkg/apis/policy/OWNERS (sig-apps-api-reviewers vs sig-apps-api-approvers)

**Comment by davidopp** [comment]
LGTM but might want to remove bprashanth

**Comment by erictune** [comment]
I don't understand how an OWNERS file could cause just federation to fail, so merging.

**Comment by kargakis** [comment]
@erictune that job is broken but has been disabled and does not block the queue: https://github.com/kubernetes/kubernetes/issues/45978#issuecomment-303011310

**Pull request merged**